### PR TITLE
Move STIPS from PYSYNPHOT to synphot/stsynphot

### LINKED ISSUE #96: Move STIPS from pysynphot to python synphot and stsynphot packages
Note that Pandeia is doing this internally as part of its next release, and I believe these packages will be better-supported in the future.

### PR DESCRIPTION
**Pull request by york-stsci**
Moves STIPS to a set of packages that will continue to be supported.

Makes STIPS compatible with pandeia >= 1.5.2.

Closes #96

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -127,21 +127,23 @@ before_install:
...
     - tar -xzvf $TEST_DATA_PATH/synphot1.tar.gz -C $TEST_DATA_PATH/synphot/
     - tar -xzvf $TEST_DATA_PATH/synphot2.tar.gz -C $TEST_DATA_PATH/synphot/
     - tar -xzvf $TEST_DATA_PATH/synphot5.tar.gz -C $TEST_DATA_PATH/synphot/
+    - tar -xzvf $TEST_DATA_PATH/synphot6.tar.gz -C $TEST_DATA_PATH/synphot/
 
     - export PYSYN_CDBS=$TEST_DATA_PATH/synphot/grp/hst/cdbs
 
     # download pandeia_data data
-    - wget -qO- https://stsci.box.com/shared/static/5j506xzg9tem2l7ymaqzwqtxne7ts3sr.gz -O $TEST_DATA_PATH/pandeia_data-1.5_wfirst.gz
+    - wget -qO- https://stsci.box.com/shared/static/7voehzi5krrpml5wgyg8bo954ew7arh2.gz -O $TEST_DATA_PATH/pandeia_data-1.5_wfirst.gz
```
Comment (robelgeda) [inline .travis.yml:143]: This should be 
`wget -qO- https://stsci.box.com/shared/static/7voehzi5krrpml5wgyg8bo954ew7arh2.gz -O $TEST_DATA_PATH/pandeia_data-1.5.2rc2_roman.gz`

File: .travis.yml [hunk 2]
```diff
@@ -127,21 +127,23 @@ before_install:
...
 
     - tar -xzvf $TEST_DATA_PATH/pandeia_data-1.5_wfirst.gz -C $TEST_DATA_PATH/
```
Comment (robelgeda) [inline .travis.yml:145]: this should be 
`tar -xzvf $TEST_DATA_PATH/pandeia_data-1.5.2rc2_roman.gz -C $TEST_DATA_PATH/`

File: environment.yml [hunk 3]
```diff
@@ -57,7 +58,7 @@ dependencies:
 
   - pip:
     # Major modules
-    - pandeia.engine==1.5
+    - pandeia.engine>=1.5.2
```
Comment (robelgeda) [inline environment.yml:61]: this should be `==`

File: .travis.yml (hunk 2 above)
Comment (york-stsci) [reply .travis.yml:145]: Done.

File: environment.yml (hunk 3 above)
Comment (york-stsci) [reply environment.yml:61]: Done.

**Review by robelgeda — APPROVED** [review]
Looks good to me, tests passing

**Pull request merged**